### PR TITLE
Page titles

### PR DESCRIPTION
**Pull request by idcuadrowolox**
## Summary
- Page Titles

## Screenshot
Screen Shot 2019-04-01 at 3 44 08 PM

## Trello Card
https://trello.com/c/qF8qPrzR/21-page-titles

### CODE REVIEW

File: maquetado/scss/_titles.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+@import 'colors';
+
+.title {
+  color: $black;
+  font-size: 25px;
```
Comment (henryzarza) [inline maquetado/scss/_titles.scss]: Use the variable `text-big`

File: maquetado/scss/_titles.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  height: 100px;
```
Comment (henryzarza) [inline maquetado/scss/_titles.scss]: Use the variable `title-decoration-height`

File: maquetado/scss/_titles.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  max-height: 100%;
+  min-width: 75px;
```
Comment (henryzarza) [inline maquetado/scss/_titles.scss]: I think that you don't need this, please delete it

File: maquetado/scss/_titles.scss [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  padding: 38px 56px;
```
Comment (henryzarza) [inline maquetado/scss/_titles.scss]: I recommend you change this for values in a scale determinate (each 5, 10, ...), try with `35px 55px`

File: maquetado/scss/_titles.scss [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  position: relative;
+  width: 100%;
+
+  &::before {
+    background: url(assets/triangle.svg);
```
Comment (henryzarza) [inline maquetado/scss/_titles.scss]: Change for `background-image`

File: maquetado/scss/_titles.scss [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    content: '';
+    display: block;
+    height: 100px;
```
Comment (henryzarza) [inline maquetado/scss/_titles.scss]: Change this for: `title-decoration-height` or `100%`

File: maquetado/scss/_titles.scss [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    left: 0px;
+    position: absolute;
+    top: 0px;
+    width: 75px;
```
Comment (henryzarza) [inline maquetado/scss/_titles.scss]: Change this for `tech-logo-size` variable

File: maquetado/scss/_titles.scss [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+  padding: 38px 56px;
+  position: relative;
+  width: 100%;
+
+  &::before {
+    background: url(assets/triangle.svg);
+    content: '';
+    display: block;
+    height: 100px;
+    left: 0px;
```
Comment (henryzarza) [inline maquetado/scss/_titles.scss]: Change by `0` without units, check the style guide 😉

File: maquetado/scss/_titles.scss [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+    position: absolute;
+    top: 0px;
```
Comment (henryzarza) [inline maquetado/scss/_titles.scss]: The same here

**Review by henryzarza — CHANGES_REQUESTED** [review]
Excellent job only a few comments and go! I recommend you create a file called `_variables.scss` and to add all the variables of this [card](https://trello.com/c/i94tzZaj/11-setup-del-proyecto)

File: maquetado/scss/_titles.scss (hunk 4 above)
Comment (idcuadrowolox) [reply maquetado/scss/_titles.scss]: I did try with `40px 55px`

**Review by henryzarza — CHANGES_REQUESTED** [review]
And `_variables.scss` file?

**Review by henryzarza — APPROVED** [review]

**Pull request merged**